microcosm_flask/audit.py

```diff
@@ -168,7 +172,13 @@ def to_dict(self):
     def log(self, logger):
         if self.status_code == 500:
             # something actually went wrong; investigate
-            logger.warning(self.to_dict())
+            dct = self.to_dict()
+
+            if current_app.debug or current_app.testing:
+                message = dct.pop("message")
```

Comment:
In debug/test, log errors so that:
 - We have a clear message
 - We have a stack trace
 - We pass everything else as extra